docker/kernel/httploader/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM golang:alpine AS build
+
+RUN apk --no-cache add build-base git bzr mercurial gcc ca-certificates
+
+ADD ./httploader /src
```

Comment:
Add a .dockerignore file containing Dockerfile to avoid copying the Dockerfile itself into /src.

Reply:
You're right, my fault. :)
Resolving!